LocationServices/LocationServices/Constants/StringConstants.swift

```diff
@@ -48,7 +48,7 @@ enum StringConstant {
     
     
     // strings:
-    static let direction = "Direction"
+    static let direction = "Directions"
     static let directions = "Directions"
```

Comment:
If you no longer need a string that says "Direction", we should just use the existing `directions` string directly below this and delete `direction`.

Reply:
fixed